java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/util/HexString.java

```diff
@@ -4,7 +4,7 @@
 
 /** Utility method to convert hexadecimal string from/to longs and byte arrays.
  *
- * @author Andreas Wundsam <andreas.wundsam@bigswitch.com>
+ * @author Andreas Wundsam {@literal <}andreas.wundsam@bigswitch.com{@literal >}
```

Comment:
Weird I'm using that author tag *everywhere* and haven't seen a problem with it yet....